Aandachtspunten.md

```diff
@@ -1,9 +1,28 @@
 ## Aandachtspunten
```

Comment:
Het lijkt erop dat je de juiste structuur hebt aangehouden voor je modellen en dat je gebruik maakt van Dependency Injection via de constructor, wat uitstekend is!